lib/cc/engine/analyzers/javascript/main.rb

```diff
@@ -29,7 +29,7 @@ def run
           end
 
           def mass_threshold
```

Comment:
Maybe now is not the time, but it seems like these per-language `main` classes are a good case for inheritance?